system_tests/espresso_test.go

```diff
@@ -1,352 +0,0 @@
-package arbtest
```

Comment:
@ImJeremyHe @sveitser any objections to deleting this test? I think that the E2E test does strictly more, and it isn't trivial to mock the namespace proofs that we fetch from hotshot correctly